scripts/environment.py

```diff
@@ -216,12 +223,25 @@ def deployEmptyProxy(self):
         proxy = nProxy.deploy(emptyProxyImpl.address, bytes(), {"from": self.deployer})
         return Contract.from_abi("Proxy", proxy.address, EmptyProxy.abi)
 
-    def upgrade_sNOTE(self):
+    def deployBalancerToken(self):
+        return MockERC20.deploy("Mock Balancer Token", "BAL", 18, 0, {"from": self.deployer})
+
+    def upgrade_sNOTE(self, treasuryManager):
+        self.balancerMinter = MockBalancerMinter.deploy(self.bal, {"from": self.deployer})
```

Comment:
Instead of mocking these we should use the real ones:
    "LiquidityGauge": "0x40ac67ea5bd1215d99244651cc71a03468bce6c0",
    "BalancerMinter": "0x239e55F427D44C3cc793f49bFB507ebe76638a2b",
    
I think we should update the environment config to point to the real sNOTE pool and then just run these tests on a more recent block. It'll be more realistic and ensure we catch any other edge cases